packages/drivers/http/src/http.js

```diff
@@ -75,9 +77,10 @@ const getDriver = (store) => {
   }
 
   // custom helpers
-  driver.setAuthorization = (authorization) => { innerHeaders.Authorization = authorization }
   driver.setOptions = (options) => { innerOptions = options }
   driver.setCredentials = (credentials) => { innerOptions = { ...innerOptions, credentials } }
+  driver.setAuthorization = (authorization) => { innerHeaders.Authorization = authorization }
```

Comment:
To avoid doing the `if` in every request. (and the delete)
Why don't you run the delete here ?

```
driver.setAuthorization = (authorization) => {
  if (authorization) innerHeaders.Authorization = authorization
  else delete innerHeaders.Authorization
}
```

Reply:
si je fais ca a ce moment la, le `innerOptions` va surcharger le `innerHeader` 😞 et donc comment passer l'information qu'on veut surcharger par `undefined` ?

Reply:
Then we remove `innerHeader` and do something like this ?
```
driver.setOptions = (options) => { innerOptions = { ...options, headers: { ...options.headers } } }
driver.setAuthorization = (authorization) => {
  const headers = { ...innerOptions.headers, Authorization: authorization }
  if (!authorization) delete newHeaders.Authorization

  return driver.setOptions({ ...innerOptions, headers } })
}
```